scripts/build/builders/efr32.py

```diff
@@ -230,7 +237,12 @@ def __init__(self,
             # case for pre-installed images), use it directly.
             sdk_path = shlex.quote(os.environ['GSDK_ROOT'])
             self.extra_gn_options.append(f"efr32_sdk_root=\"{sdk_path}\"")
-            self.extra_gn_options.append(f"openthread_root=\"{sdk_path}/util/third_party/openthread\"")
+            if not enable_wifi:
```

Comment:
why this condition? This only sets an already existing path variable. Even if wifi is used it doesn't matter that a path is specified for the openthread_root

Reply:
there is no openthread_root declared for the 917 soc front so added this condition